Remove node-authorization (#10439)
kubernetes/kops: docs/node_authorization.md

### **Node Authorization Service**

**The node authorization service is deprecated.**

> In kOps 1.19 using Kubernetes 1.19 kOps will, on AWS, ignore the `nodeAuthorization` field of the cluster spec and

Reviewer: Seems a little unclear.
```suggestion
As of kOps 1.19 with AWS and Kubernetes 1.19, the `nodeAuthorization` field of the cluster spec will be ignored  and
```

worker nodes will obtain client certificates for kubelet and other purposes through kops-controller.

> In kOps 1.20, setting `nodeAuthorization` is forbidden on any Kubernetes version and you will have to upgrade to kubernetes 1.19+ to retain this feature.

Reviewer: Again, not sure I understand.
```suggestion
As of kOps 1.20, setting `nodeAuthorization` is forbidden for any Kubernetes version and is replaced by the bootstrapping using kops-controller.
```
